login.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+	<link href="login.css" rel="stylesheet"/>
+	
+	<!-- jQuery library, then Popper JS, then JavaScript -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+
+    <title>MyMood Database Interface</title>
+  </head>
+  <body>
+  <!-- Navigation Bar -->
+    <nav class="navbar navbar-expand-sm navbar-dark bg-primary" id="mainNav">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="database.html">MyMood</a>
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="help.html">Help</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="aboutus.html">About Us</a>
+          </li>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
+              Light Theme
+            </a>
+            <div class="dropdown-menu">
+              <a class="dropdown-item" href="#">Dark Theme</a>
+            </div>
+          </li>
+          <li class="nav-item active">
+            <a class="nav-link" href="login.html">Login</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+	
+	<!-- Login Box -->
+	<div class="container">
+	  <div class="row">
+	    <div class="col-md-offset-5 col-md-3">
+		  <div class="loginForm">
+		    <h4 class="headingLogin">Login</h4>
+			<input type="text" id="username" class="form-control input-sm" placeholder="Username"/>
+			<p class="invalidUser">Invalid Username.</p>
+			<br>
+			<input type="password" id="password" class="form-control input-sm" placeholder="Password"/>
+			<p class="invalidPass">Invalid Password.</p>
+			<br>
+			<div class="alignment">
+			  <button id="submit" type="submit" class="btn btn-primary">Submit</button>
+			</div>
+		  </div>
+	    </div>
+      </div>
+	</div>
+	
+	<!-- Login Credentials -->
+	<script>
+	$(document).ready(function() {
+      $('#submit').click(function() {
+		event.preventDefault(); // prevent PageReLoad
+		var ValidUsername = $('#username').val() === 'admin'; 
+		var ValidPassword = $('#password').val() === 'password';
+		if (ValidUsername === true && ValidPassword === true) { 
+		  window.location = "database.html"; 
+		}
+		if (ValidUsername === false){

```

Comment:
to be consistent, there should be a space after if (...)

Like so:
if (ValidUsername === false) {